edit/resources.cattle.io.backup.vue

```diff
@@ -49,29 +49,47 @@ export default {
 
     const hash = await allHash({
       secrets:      this.$store.dispatch('cluster/findAll', { type: SECRET }),
-      resourceSets: this.$store.dispatch('cluster/findAll', { type: BACKUP_RESTORE.RESOURCE_SET }),
+      resourceSet: this.$store.dispatch('cluster/find', { type: BACKUP_RESTORE.RESOURCE_SET, id: 'rancher-resource-set' }),
```

Comment:
I get that we removed the option for choosing the resource set in the UI, but on edit it should still grab the one the spec says instead of assuming it's `rancher-resource-set`, or you're going to overwrite what they had set it to silently.

Reply:
updated to find by the name specified in the backup spec, if available.